app/models/link_check_report.rb

```diff
@@ -0,0 +1,18 @@
+class LinkCheckReport
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  embeds_many :links
+
+  accepts_nested_attributes_for :links
+
+  field :batch_id, type: Integer
+  field :status, type: String
+  field :link_reportable_type, type: String
+  field :link_reportable_id, type: Integer
```

Comment:
I wonder if we need the same polymorphic relationship with link check reports in manuals publisher.

Are we going to be checking links for more than manuals? Perhaps @thomasleese would know.

Reply:
@lukemorton Seb thinks that there are a few different types, so think link_reportable_type is needed